<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="DESKTOP-OSVRB79" failures="1" tests="2" name="Default test(failed)" time="0.117" errors="0" timestamp="2023-12-26T13:12:29 IST">
  <testcase classname="testNg_FailedXML_Serial_Parallel.Failed_xml" name="e" time="0.017">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testNg_FailedXML_Serial_Parallel.Failed_xml.e(Failed_xml.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- e -->
  <testcase classname="testNg_FailedXML_Serial_Parallel.Failed_xml" name="c" time="0.008"/>
  <testcase classname="testNg_FailedXML_Serial_Parallel.Failed_xml" name="d" time="0.0">
    <ignored/>
  </testcase> <!-- d -->
  <testcase classname="testNg_FailedXML_Serial_Parallel.Failed_xml" name="a" time="0.0">
    <ignored/>
  </testcase> <!-- a -->
  <testcase classname="testNg_FailedXML_Serial_Parallel.Failed_xml" name="b" time="0.0">
    <ignored/>
  </testcase> <!-- b -->
</testsuite> <!-- Default test(failed) -->
